platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -1249,13 +1250,22 @@ public long estimatedSize() {
      */
     @Override
     public void serialize(final SerializableDataOutputStream out, final Path outputDirectory) throws IOException {
-        final RecordAccessor<K, V> detachedRecords = pipeline.detachCopy(this, outputDirectory);
-        assert detachedRecords.getDataSource() == null : "No data source should be created.";
+        final long start = System.currentTimeMillis();
+        //        pipeline.snapshot(this, outputDirectory);
```

Comment:
If it's meant to be removed, please remove it.

Reply:
It is removed in the recent versions